package.json

```diff
@@ -15,16 +15,22 @@
     "styled-components": "^5.0.1"
   },
   "devDependencies": {
+    "@types/jest": "^25.1.3",
     "@types/node": "^13.7.7",
     "@types/react": "^16.9.23",
     "@types/react-dom": "^16.9.5",
+    "@types/react-test-renderer": "^16.9.2",
     "@types/styled-components": "^5.0.1",
+    "jest": "^25.1.0",
+    "react-test-renderer": "^16.13.0",
+    "ts-jest": "^25.2.1",
     "typescript": "^3.8.3"
   },
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
+    "test:watch": "jest --watch"
```

Comment:
There's a lot of variety to run tests, such as `--watch`, `--notify`, `--updateSnapshots`. So we should put options manually such like runing `npm test -- --watch` or (with `--notify`) when we want to test code in watch mode.

```suggestion
```